### ISSUE TITLE
Any doc here?

### ISSUE THREAD

**Issue by odil24** [sergcpp/RayDemo#2]
Is there have any documentation on how to use the engine?

**Comment by sergcpp** [comment]
Not really. My plan is a little bit different. This repo is merely a sample application, I only need it to test some scenes in CI builds to catch regression.
- What I want to do in the nearest month is merge my two hobby projects into one. So that there would be a fully realtime engine for fast iteration and batch screenshot rendering and offline engine for high quality final renderings. With a seamless switch between the two, just like Eevee and Cycles work in Blender.
- After that I will start to work on a plugin for Sketchup. I think it does not make much sense to compete with what Blender already has, Eevee and Cycles are almost perfect. But there is a place I can take in other modeling pakages like Sketchup. There is no good free renderer there with a combination of realtime and offline engine.

**Comment by odil24** [comment]
> Not really. My plan is a little bit different. This repo is merely a sample application, I only need it to test some scenes in CI builds to catch regression.
> 
> * What I want to do in the nearest month is merge my two hobby projects into one. So that there would be a fully realtime engine for fast iteration and batch screenshot rendering and offline engine for high quality final renderings. With a seamless switch between the two, just like Eevee and Cycles work in Blender.
> * After that I will start to work on a plugin for Sketchup. I think it does not make much sense to compete with what Blender already has, Eevee and Cycles are almost perfect. But there is a place I can take in other modeling pakages like Sketchup. There is no good free renderer there with a combination of realtime and offline engine.

So, are you dont want continue developing for Blender I guess?

**Comment by sergcpp** [comment]
> So, are you dont want continue developing for Blender I guess?

I definitely want to have some exporter script for internal use and testing (it literally takes me 2 days to manually export every big scene). But no, I think no one needs my toy renderer in Blender when far better build-in one exists.

**Comment by odil24** [comment]
> > So, are you dont want continue developing for Blender I guess?
> 
> Я определенно хочу иметь какой-то скрипт экспортера для внутреннего использования и тестирования (мне буквально требуется 2 дня, чтобы вручную экспортировать каждую большую сцену). Но нет, я думаю, что никому не нужен мой игрушечный рендерер в Blender, когда существует гораздо лучший встроенный.

Maybe, but I tested out raydemo and it is very faster also with CPU mode than Cycles in Blender

**Comment by sergcpp** [comment]
> Maybe, but I tested out raydemo and it is very faster also with CPU mode than Cycles in Blender

Чтож, я сам пока не сравнивал и не знаю почему так, возможно какие-то фичи делают его медленнее. Насколько я знаю Cycles использует Embree, kernel'ы написанные Intel. Они должны быть быстрее того, что я написал.

**Comment by odil24** [comment]
> > Maybe, but I tested out raydemo and it is very faster also with CPU mode than Cycles in Blender
> 
> Чтож, я сам пока не сравнивал и не знаю почему так, возможно какие-то фичи делают его медленнее. Насколько я знаю Cycles использует Embree, kernel'ы написанные Intel. Они должны быть быстрее того, что я написал.


https://user-images.githubusercontent.com/64298012/212741286-8a608d7f-4dd3-4b11-ab82-f2b1452ff9cf.mp4
